libs/designer-ui/src/lib/dynamicallyaddedparameter/dynamicallyaddedparameter.less

```diff
@@ -0,0 +1,102 @@
+.msla-dynamic-added-param-header {
```

Comment:
It doesn't unless you import it here: https://github.com/Azure/LogicAppsUX/blob/main/libs/designer-ui/src/lib/styles.less

Reply:
thanks! Imported it in styles.less for now. I haven't yet verified that the styling is what we want though, because some of the elements have changed and I copied the styles from BPMUX repo

Reply:
update: going through all the styles in this file and making sure they're good